### DISCUSSION TITLE
I have 4 achievements but 1 not appears

### DISCUSSION THREAD

**Discussion by SenaJp** [community/community#67107] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

## In my profile show only 3:
![Profile preview](https://github.com/community/community/assets/86388032/b9500bdf-e46d-4bed-80d4-75b1efd4cb51)

## But when i open the achievements show all:
![Achievements](https://github.com/community/community/assets/86388032/e7c62dd8-3f44-4167-b391-64596c891f82)

**How to solve this?**

**Comment by VedantiBhoyar** [comment]
Clear Browser Cache: Cached data can sometimes cause display issues. Clear your browser's cache and cookies, then reload the page and check if the problem is resolved.

Check for Updates: Ensure that your web browser is up to date. Outdated browsers can sometimes have compatibility issues that cause display problems.

Browser Extensions: Certain browser extensions or add-ons can interfere with website functionality. Disable all extensions temporarily to see if any of them are causing the issue.

**Reply by SenaJp** [reply]
I tested in another computer and anothers browsers but not appears. If you access my profile u can se all achievements?

**Comment by radar-blip** [comment, marked as answer]
When you open the achievements,
you can click on the achievement you want to show by clicking the eye icon to hide or show the achievement.
![show](https://github.com/community/community/assets/39196767/38506e94-9e31-4c33-a838-328d46a866a0)

**Reply by SenaJp** [reply]
Its works! I selected it to hide from the profile and then I selected it to be displayed and it appeared there. **Thank you very much!**

**Reply by radar-blip** [reply]
You're welcome. Happy to help!

**Comment by Robyn37** [comment]
Love
